### PR TITLE
Lists and Modal Component added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Lists

- Normal List
- Stacked List

### Modal
Screenshot (9)
Screenshot (10)
Screenshot (11)
l

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (vaibhav18matere) [inline .vscode/settings.json:3]: why we need this setting.json?

File: component/list.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.normal-list{
+    display: flex;
+}
+
+#stacked-list li{
+    width: fit-content;
+    border-radius: 1rem;
+    display: flex;
```
Comment (vaibhav18matere) [inline component/list.css:12]: you could have used default property if already added for `display:flex`

**Review by vaibhav18matere — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (sakleprajwal) [reply .vscode/settings.json:3]: Yes you can directly remove thi setting.json

**Review by sakleprajwal — APPROVED** [review]
Added a comment. Everything else looks good to go.

**Review by harshkotkar5557 — APPROVED** [review]
Awesome everything looks good

File: .vscode/settings.json (hunk 1 above)
Comment (Mahi24-prog) [reply .vscode/settings.json:3]: ok

**Pull request merged**